src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private int threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    private Node<K, V>[] table = (Node<K,V>[]) new Node[INITIAL_CAPACITY];
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        putVal(key, value, table);
     }
 
     @Override
     public V getValue(K key) {
+        Node<K, V> currentNode = gettingNodeByKey(key);
+        while (currentNode != null) {
+            if (Objects.equals(currentNode.key, key)) {
+                return currentNode.value;
+            }
+            currentNode = currentNode.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int hash(Object key) {
```

Comment:
Private ones (methods, classes) should be after public ones.
Not fixed:
![image](https://user-images.githubusercontent.com/20326858/202146872-ef01858b-b196-4ba2-901b-3225e0f84dff.png)